### ISSUE TITLE
Debezium source connector error with Postgresql

### ISSUE THREAD

**Issue by haijianyang** [apache/pulsar#14397]
**Describe the bug**
Pulsar 2.9.1
PG 14.1

Following the operation of the document(https://pulsar.apache.org/docs/en/io-cdc-debezium/#example-of-postgresql), the following error occurred:
<img width="934" alt="image" src="https://user-images.githubusercontent.com/19967151/154911165-6b5918e9-b8f2-4f9a-a0b0-f7c63738597a.png">
```bash
2022-02-21T07:49:22,278+0000 [function-timer-thread-8-1] ERROR org.apache.pulsar.functions.runtime.RuntimeSpawner - public/default/debezium-postgres-source Function Container is dead with following exception. Restarting.
java.lang.IllegalArgumentException: Pulsar service URL for History Database not provided.
	at org.apache.pulsar.io.debezium.DebeziumSource.open(DebeziumSource.java:85) ~[?:?]
	at org.apache.pulsar.functions.instance.JavaInstanceRunnable.setupInput(JavaInstanceRunnable.java:757) ~[org.apache.pulsar-pulsar-functions-instance-2.9.1.jar:2.9.1]
	at org.apache.pulsar.functions.instance.JavaInstanceRunnable.setup(JavaInstanceRunnable.java:232) ~[org.apache.pulsar-pulsar-functions-instance-2.9.1.jar:2.9.1]
	at org.apache.pulsar.functions.instance.JavaInstanceRunnable.run(JavaInstanceRunnable.java:260) ~[org.apache.pulsar-pulsar-functions-instance-2.9.1.jar:2.9.1]
	at java.lang.Thread.run(Thread.java:829) [?:?]
```

**Comment by gaozhangmin** [comment]
It's already fixed by #12145  @haijianyang

**Comment by haijianyang** [comment]
LGTM
